samples/als-sink/README.md

```diff
@@ -0,0 +1,5 @@
+# als-sink
```

Comment:
Can you query metrics rather than using access log?  Which should be simpler.

Reply:
https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider-EnvoyHttpGrpcV3LogProvider
```
 "filterStateObjectsToLog": [
            "wasm.upstream_peer",
            "wasm.upstream_peer_id",
            "wasm.downstream_peer",
            "wasm.downstream_peer_id"
```
This metadata about peer envoy, along with some more details, is exchanged by tcp and http metadata exchange filters running on client and server envoy. Using the above shared config, these metadata fields get reported by envoy to grpc als sink service. Can please point me to what metrics can I use to confirm this MX? I know only als service method.

Reply:
IIRC, there's no such metrics, maybe @douglas-reid know more.

Reply:
For metadataexchange, it sends node meta to upstream, So i mean you can check the source_workload in upstream

Reply:
Big +1 to @hzxuzhonghu 's comment. We should be testing end user behavior, not low level implementation details, in our e2e tests

Adding ALS tests is useful for other reasons, but should not be used to test this change IMO.